puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -1,22 +1,38 @@
 # @api private
 class slave::postgresql {
-  Postgresql_psql {
-    cwd => '/',
-  }
+  # only CentOS slaves are used to run unit tests
+  if $::osfamily == 'RedHat' {
```

Comment:
The modern way of using facts is
```suggestion
  if $facts['osfamily'] == 'RedHat' {
```

When using non-legacy facts it looks like:
```suggestion
  if $facts['os']['family'] == 'RedHat' {
```